### PR TITLE
bani-customerportal-2022-12-29

### PR DESCRIPTION
**Pull request by Bani2791**
## **1. Customer portal > Dashboard**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-dashboard/

## **2. Customer portal > Customer Portal**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-cdr/

## **3. Customer portal > Breakout**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-breakout/

## **4. Customer portal > Authentication**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-authentication/

## **5. Customer portal > DID**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-did/

## **6. Customer portal > Invoice**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-invoice/

## **7. Customer portal > CLI**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-cli/

## **8. Customer portal > Latest Calls**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-latest-calls/

## **9. Customer portal > User registration**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-user-reg/

## **10. Customer portal > User registration**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-user-reg/

## **11. Customer portal > PBX > Conference**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-conference/

## **12. Customer portal > PBX > IVR**

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-ivr/

### CODE REVIEW

File: docs/customer-portal/cp-dashboard.md [hunk 1]
```diff
@@ -1,44 +1,46 @@
...
-+ **Account Balance** Displays the current account balance and a link to make a payment using a credit card or PayPal. 
++ **Account Balance** Displays the current account balance and a link to make a payment using a credit card or PayPal.
 
 + **Account** Use this to view your Company Profile, view and update your personal Profile, Change Password or Language, or Log Out.
 
-
 !!! warning "Account Balance functions"
-    The "Click to Make Payment" link is currently in development. Please navigate to **Payment** on the left-hand menu for this functionality. 
-
+    The "Click to Make Payment" link is presently under development. Please navigate to **Payment** on the left-hand menu for this functionality.
```
Comment (syedrizwansrs) [inline docs/customer-portal/cp-dashboard.md:20]: This feature is live and ready for use, please remove this Warning

File: docs/customer-portal/cp-invoice.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 # Invoice
-This section lists each **Invoice** that is generated for your account. The following functions are available:
 
-+ **Vice Invoice**: Click this to display the invoice in a webpage.
-+ **Download as PDF**: Click this to download a copy of the invoice to your computer. 
+This section lists each generated **Invoice** for your account. The following functions are available:
 
-![alt text][cp-invoice] 
++ **Vice Invoice:** Click here to display the invoice on a webpage.
```
Comment (syedrizwansrs) [inline docs/customer-portal/cp-invoice.md:5]: Vice Invoice, not sure if you meant to add vice, please check

**Review by syedrizwansrs — CHANGES_REQUESTED** [review]
Please see the comments.
Thanks

File: docs/customer-portal/cp-dashboard.md (hunk 1 above)
Comment (Bani2791) [reply docs/customer-portal/cp-dashboard.md]: removed the warning

please check

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-dashboard/

File: docs/customer-portal/cp-invoice.md (hunk 2 above)
Comment (Bani2791) [reply docs/customer-portal/cp-invoice.md]: Its **View Invoice**: corrected it

please check:

https://bani-customerportal-2022-12-29--connexcs-docs.netlify.app/customer-portal/cp-invoice/

**Review by syedrizwansrs — APPROVED** [review]

**Pull request merged**